js/menujs.js

```diff
@@ -9,7 +10,7 @@ hamburger.addEventListener('click', (event) => {
   hamburger.classList.toggle('showMobil');
 });
 
-console.log(arrAncherTag);
+// console.log(arrAncherTag);
 
 arrAncherTag.forEach((a) => {
```

Comment:
- [ ] Please make reference to the line `101` to see my detailed review on wy you should remove the `console.log` comment 👍